srcpkgs/grilo/patches/fix-pc-file.patch

```diff
@@ -0,0 +1,11 @@
+--- a/meson.build
```

Comment:
Add a description for why this patch is necessary.

Reply:
will do. unfortunately, your patch didn't work for cross-compilation targets. at least, mine did.

did i do something wrong?

Reply:
```
Variable 'datadir' not defined in '/usr/aarch64-linux-gnu/usr/lib/pkgconfig/grilo-0.3.pc'
```

Reply:
Yeah, just replace it with `join_paths('${datadir}', 'gir-1.0')`.

Reply:
fixed, thanks.